docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml

```diff
@@ -38,16 +44,24 @@ code: |
     progressor.increment()  # 1
 
   progressor.increment()  # 2
-
+  show_docket_sheets_info = True
```

Comment:
`show_docket_sheets_info` doesn't seem to be used anywhere

Reply:
I'm not completely sure, but I think this may come into play in the rest of the sealing form - the next file. I don't have time to check at the moment, though.